src/components/TransactionItemComponent/index.tsx

```diff
@@ -13,64 +14,73 @@ import TagCell from './DataCeils/TagCell';
 import TotalCell from './DataCeils/TotalCell';
 import TypeCell from './DataCeils/TypeCell';
 
-function TransactionItemComponent({transactionItem, isLargeScreenWidth, isSelected}: {transactionItem: Transaction; isLargeScreenWidth: boolean; isSelected: boolean}) {
+function TransactionItemComponent({
+    transactionItem,
+    shouldUseNarrowLayout,
+    isSelected,
+    showTooltip,
```

Comment:
```suggestion
    shouldShowTooltip,
```

This name abides by our code of conduct

Reply:
Here I am not convinced because in the project we already use `showToltip` for example in:
`TextWithIconCell.tsx`
but also `shuldShowToltip`:
`TextWithTooltip`

Reply:
Coming from: 
https://github.com/Expensify/App/blob/main/contributingGuides/STYLE.md#boolean-variables-and-props

Reply:
I believe that we shouldn't use `showToltip` as a boolean variable name.